app/channel.go

```diff
@@ -1293,11 +1293,11 @@ func (a *App) GetChannelMember(channelId string, userId string) (*model.ChannelM
 }
 
 func (a *App) GetChannelMembersPage(channelId string, page, perPage int) (*model.ChannelMembers, *model.AppError) {
-	result := <-a.Srv.Store.Channel().GetMembers(channelId, page*perPage, perPage)
```

Comment:
Here you can just `return a.Srv.Store.Channel().GetMembers(channelId, page*perPage, perPage)`